src/pages/Messages.tsx

```diff
@@ -1,5 +1,68 @@
+import { Box, Button, Divider, List, Stack, Typography } from "@mui/material";
+import ChatIcon from "@mui/icons-material/Chat";
+import IconButton from "@mui/material/IconButton";
+import SearchBar from "../components/Messages/SearchBar";
+import MessagesList from "../components/Messages/MessagesList";
+
+const styles = {
+  button: {
+    borderRadius: 10,
+    marginTop: 1,
+  },
+  messagesHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    paddingTop: 2,
+    paddingRight: 2,
+    paddingLeft: 2,
+    paddingBottom: 0,
+  },
+  messageListContainer: {
+    width: "30%",
+  },
+  root: {
+    width: "100%",
+  },
+  selectMessageContainer: {
+    alignItems: "flex-start",
+    display: "flex",
+    flexDirection: "column",
+    margin: "auto",
+    width: "50%",
+  },
+};
+
 const Messages = () => {
-  return <h1 style={{ margin: "auto" }}>Messages</h1>;
+  return (
+    <Stack
+      direction="row"
+      sx={styles.root}
+      divider={<Divider orientation="vertical" />}
```

Comment:
Nice use of the divider attribute, I'd like to implement the `Stack` as the parent element in other pages as well.